lib/chainEnv.ts

```diff
@@ -1,4 +1,9 @@
 export const mainnet = (): boolean => process.env.NEXT_PUBLIC_ENV === 'mainnet';
 
+export const optimism = (): boolean =>
```

Comment:
aside: do these need to be function calls? I think this could be statically known at build-time

Reply:
good call